src/writing-your-own-application/alloc.md

```diff
@@ -0,0 +1,53 @@
+# Alloc
+
+In `no_std`, we need to rely on `alloc` crate, which provides heap allocation support. To use `alloc` in `no_std` environment:
```

Comment:
I think this needs rewording. We do not _need_ to rely on `alloc`, it is only required if people which to perform heap allocations.